modules/environment-roles/templates/tdr_jenkins_read_params.json.tpl

```diff
@@ -17,7 +17,9 @@
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/backend_checks_client/secret",
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/realm_admin_client/secret",
         "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/configuration_properties",
-        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/user_admin_client/secret"
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/user_admin_client/secret",
+        "arn:aws:ssm:eu-west-2:${account_id}:parameter/${environment}/keycloak/govuk_notify/api_key",
```

Comment:
Just to check - why does Jenkins need permission to read the Notify API key? Does it copy the key to each environment?

Reply:
This gets attached to the terraform iam role to give it permission to create the ssm params in each of the environments

Reply:
Oh I see - this creates the parameter within SSM, but you have to set it manually in each account?

Reply:
Correct. Sorry should have made that clear.